client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -362,6 +372,26 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: cacheIcon,
 		},
+		{
+			name: 'visitSite',
+			label: __( 'Visit site' ),
```

Comment:
I would like to add "homepage" or "front" to the command name to be more explicit.

Reply:
That sounds good. 👍

Reply:
Thanks for the changes!  ❤️